### PR TITLE
08-28-2023 console release notes

### PR DESCRIPTION
**Pull request by danieltprice**
Preview:
https://neon-next-git-dprice-08-28-2023-console-release-neondatabase.vercel.app/docs/release-notes/2023-08-28-console

### CODE REVIEW

**Review by duskpoet — APPROVED** [review]

**Comment by danieltprice** [comment]
> We also need to include release notes from release which rolled back on Friday if any

Thank you. I see that we reverted and then added the changes for 08/24 back in this PR. I have added a couple of items from the 08/24 release.

File: content/docs/introduction/billing.md [hunk 1]
```diff
@@ -188,6 +188,11 @@ The [Pricing](https://neon.tech/pricing) page on the Neon website provides a cal
 
 ### Pro Plan Cost Estimator
 
-The **Billing** page in the Neon Console provides a **Pro Plan Cost Estimation** tool, which estimates costs based on your current project usage. If you have ever wondered, "How much would my monthly cost be if I upgraded to the Neon Pro plan today?", this tool provides the answer. To access it, navigate to the **Billing** page via the sidebar of the Neon Console, and select **Estimate costs**.
+The **Billing** page in the Neon Console provides a **Pro Plan Cost Estimation** tool, which estimates costs based on your current project usage. If you have ever wondered, "How much would my monthly cost be if I upgraded to the Neon Pro plan today?", this tool provides the answer. To access it, navigate to the **Billing** page via the sidebar of the Neon Console, and select **Open cost calculator**.
+
+The tool displays cost estimates only after following conditions are met:
+
+- Your account has been registered for at least one month.
```
Comment (duskpoet) [inline content/docs/introduction/billing.md:195]: That's how it was described in the issue, but I guess the second kind of implies the first one?

**Review by duskpoet — APPROVED** [review]
UI part looks good, couple nits

File: content/docs/introduction/billing.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/introduction/billing.md]: Good point. Revised.

File: content/release-notes/2023-08-28-console.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+### Fixes & improvements
+
+- Control Plane: The [Availability Checker](/docs/reference/glossary#availability-checker) now uses the smallest Neon compute size (.25 Compute Units) to verify that a project's computes can start and read and write data. This change allows the Availability Checker to use pre-started computes more often, which shortens availability check operation time and reduces the impact of availability checks on infrastructure and compute costs.
+- Control Plane: Removed a write check that was performed on a read-only compute instance. A write check is only necessary for read-write compute instances.
```
Comment (piercypixel) [inline content/release-notes/2023-08-28-console.md:4]: This line can be removed altogether IMO.

This one was a fix for a bug that reproduced in some rare cases, like project migrations between pageservers via our internal Admin API, most probably not visible to our customers at all.

**Review by piercypixel — APPROVED** [review]

**Comment by piercypixel** [comment]
🚀

File: content/release-notes/2023-08-28-console.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (piercypixel) [reply content/release-notes/2023-08-28-console.md:4]: This line can be removed altogether IMO.

This one was a fix for a bug that reproduced in some rare cases, like project migrations between pageservers via our internal Admin API, most probably not visible to our customers at all.

[the remark this answers is not in the document]
Comment (danieltprice) [reply content/release-notes/2023-08-28-console.md:4]: Thank you. Removed.

**Review by piercypixel — APPROVED** [review]

**Review by duskpoet — APPROVED** [review]

**Pull request merged**